cmd/gluster-virtblock/main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	gvb "github.com/gluster/gluster-csi-driver/pkg/gluster-virtblock"
+	"github.com/gluster/gluster-csi-driver/pkg/gluster-virtblock/config"
```

Comment:
I feel , either virtualblock or `vb` suite better than virtblock.

Reply:
Also considering glusterfs and gluster vb driver use same config params of a GD2 cluster, `config` an outside library for both to consume once its imported.